packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -51,22 +66,62 @@ export class PlansPage {
 		await this.page.waitForLoadState( 'load' );
 	}
 
+	/* Current Plans */
+
+	/**
+	 * Clicks on the button to initiate upgrade to WordPress.com Pro plan.
+	 */
+	async upgradeToPro(): Promise< void > {
+		const locator = this.page.locator( selectors.upgradeToProButton );
+		await Promise.all( [
+			this.page.waitForNavigation( { url: /.*checkout.*/ } ),
+			locator.click(),
+		] );
+	}
+
+	/**
+	 * Clicks on the Show/Hide Plan Comparison button at the bottom of the table.
+	 *
+	 * @param {PlansComparisonAction} action Action to perform on the Plans comparison button.
+	 */
+	async clickPlanComparisonActionButton( action: PlansComparisonAction ): Promise< void > {
```

Comment:
100% optional suggestion: for here, what do you think of using that pattern you came up with where you have two methods which each guarantee their ending state? Like the [two methods here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts#L161)?

I've just been a huge fan of that design ever since you proposed it! 😄